Explorer/Assets/Scripts/ECS/StreamableLoading/DeferredLoading/DeferredLoadingSystem.cs

```diff
@@ -0,0 +1,91 @@
+using Arch.Core;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using ECS.Abstract;
+using ECS.Prioritization.Components;
+using ECS.StreamableLoading.AssetBundles;
+using ECS.StreamableLoading.AssetBundles.Manifest;
+using ECS.StreamableLoading.Common.Components;
+using ECS.StreamableLoading.Textures;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Unity.Collections.LowLevel.Unsafe;
+using UnityEngine;
+
+namespace ECS.Prioritization.DeferredLoading
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [UpdateAfter(typeof(PrepareAssetBundleManifestParametersSystem))]
+    [UpdateBefore(typeof(LoadAssetBundleManifestSystem))]
+    [UpdateBefore(typeof(LoadTextureSystem))]
+    [UpdateBefore(typeof(LoadAssetBundleSystem))]
+    public partial class DeferredLoadingSystem<TAsset, TIntention> : BaseUnityLoopSystem where TIntention: struct, ILoadingIntention
+    {
+        private static readonly QueryDescription CREATE_LOADING_REQUEST = new QueryDescription()
+                                                                         .WithAll<TIntention>()
```

Comment:
```suggestion
                                                                         .WithAll<TIntention, PartitionComponent>()
```
Query should include whatever is being accessed